src/pyhf/tensor/jax_backend.py

```diff
@@ -32,7 +31,7 @@ def __init__(self, loc, scale):
         self.scale = scale
 
     def sample(self, sample_shape):
-        return norm.osp_stats.norm(self.loc, self.scale).rvs(
+        return osp_stats.norm(self.loc, self.scale).rvs(
```

Comment:
For reasons I don't understand, Codecov is now trying to say that this line is uncovered. :? https://codecov.io/gh/scikit-hep/pyhf/pull/1134/diff#D2-34

Reply:
it might not be covered since we don't have toys implemented to do the sampling yet.

Reply:
yeah same with above: should also be cast as jax tensors

Reply:
This should be resolved now.